### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by speakingoftime**
I added some rudimentary media queries

### CODE REVIEW

File: test2.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+please delete
```
Comment (nok-ko) [inline test2.txt:1]: I remember why this is here, but we may as well delete it. :P

**Review by nok-ko — CHANGES_REQUESTED** [review]
Makes sense, I think. Just delete that extraneous file and it will be good to merge.

File: test2.txt (hunk 1 above)
Comment (speakingoftime) [reply test2.txt]: Deleted file and pushed!

**Comment by nok-ko** [comment]
Rebase on top of `dev` and make sure the conflicts are all resolved before merging, (and the CSS looks as expected,) then we should be good to go.

**Review by nok-ko — APPROVED** [review]

**Pull request merged**